### ISSUE TITLE
False positive check

### ISSUE THREAD

**Issue by onagorodniuk** [arachnys/cabot#595]
Hello,

I have some strange behavior, sometimes different checks fails but in RAW data there is value, see screenshot:

![image](https://user-images.githubusercontent.com/11853717/35632226-c33a9faa-06ae-11e8-8bc9-8766d9e31043.png)

**Comment by JeanFred** [comment]
Just to confirm: this is a Graphite check, correct?

**Comment by onagorodniuk** [comment]
@JeanFred Yes, that's correct. Sorry for quite short description of issue.

**Comment by onagorodniuk** [comment]
@JeanFred Also hitting button 'Run check manually' gives thats behavior every time.

**Comment by onagorodniuk** [comment]
Looks like I have figure out what was the cause of the problem. From beginign of using cabot I have facing next problem: When cabot performs the graphite check it get latest data from graphite but sometimes datapoint can't be received before the check is done, so it make check failed due to host missing.

So in previous versions of cabot I have made some dirty hack, I have made time on graphite server two minutes earlier than actual time and it worked prety fine.

In current version this hack doesn't works. So I modified cabot/cabotapp/graphite.py to get previous data from graphite like that:  http://graphite.host/render?until=-2minute&from=-3minute&target=sample.metric.value&format=json

So I faced this issue because of that dirty hacks. To be clear it is because fuction validate_datapoint in cabot/cabotapp/graphite.py

```
def validate_datapoint(datapoint, mins_to_check, utcnow):
    val, timestamp = datapoint
    secs_to_check = 60 * mins_to_check
    if val is None:
        return False
    **if timestamp > (utcnow - secs_to_check):**
        return True
    else:
        return False

```
Some times `timestamp `was less than `utcnow - secs_to_check ` and this is because checks in my case was false possitive.
So probably issue can be closed. Or maybe is a good cause to discus how could be fixed problem with missing data in graphite check.
